app/src/main/java/org/fossasia/openevent/app/module/event/about/AboutEventPresenter.java

```diff
@@ -90,6 +94,19 @@ public Copyright getCopyright() {
         return copyright;
     }
 
+    @Override
+    public void deleteCopyright() {
+        copyrightRepository
+            .deleteCopyright(copyright.getId())
+            .compose(disposeCompletable(getDisposable()))
+            .compose(progressiveErroneousCompletable(getView()))
+            .subscribe(() -> {
+                getView().showCopyrightDeleted("Copyright Deleted");
+                copyright = null;
```

Comment:
Need your help here @iamareebjamal 
I am having to add this line due to which i am getting a PMD voilation due to which the build fails.
I have to add this line because once the user deletes the copyright, the API return HTTP 404 which is sent to the logger as an error. It means in the ```loadCopyright``` method, the ```onComplete``` won't run which would have assigned the ```null``` value to the copyright variable, instead ```onError``` would be triggered here 

```subscribe(loadedCopyright -> this.copyright = loadedCopyright, Logger::logError);```

Reply:
You can suppress the PMD warning